src/base/registrations/registrar.ts

```diff
@@ -213,7 +211,6 @@ async function register(name: string, owner: string, salt: Uint8Array, config: C
   const tx = await registrar(config).connect(config.signer).register(
     name, owner, ethers.BigNumber.from(salt), { gasLimit: 150000 }
   );
-  console.log("Sent", tx);
 
   await tx.wait();
   window.localStorage.clear();
```

Comment:
There's no reason to touch this file, it was fine as it was : )

Reply:
Okay cool

Reply:
Unresolved :warning:

Reply:
resolved

Reply:
It's not, just check the diff.